internal/transport/keepalive_test.go

```diff
@@ -329,14 +330,20 @@ func (s) TestKeepaliveClientClosesWithActiveStreams(t *testing.T) {
 		t.Fatalf("Server didn't return connection object")
 	}
 	defer conn.Close()
-
 	ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
 	defer cancel()
 	// Create a stream, but send no data on it.
 	if _, err := client.NewStream(ctx, &CallHdr{}); err != nil {
 		t.Fatalf("Stream creation failed: %v", err)
 	}
-
+	// Give a chance to a few pings to be sent and acked
+	time.Sleep(2 * pingTime)
+	// Check that client can still create stream
+	if _, err := client.NewStream(ctx, &CallHdr{}); err != nil {
+		t.Fatalf("Stream creation failed: %v", err)
+	}
```

Comment:
I dont think we need this now. It should be good enough to verify that we have atleast one active stream and when the server stops responding to keepalive, stream creation starts erroring.